boot/boot.js

```diff
@@ -28,6 +28,11 @@ if($tw.node) {
 	fs = require("fs");
 	path = require("path");
 	vm = require("vm");
+	try {
```

Comment:
To avoid having to modify boot.js I think this could be done in a startup module.